src/stories/GenreSelect.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import {ComponentMeta, ComponentStory} from "@storybook/react";
+
+import {useState} from "react";
+
+import GenreSelect from "../components/GenreSelect";
+
+import {GenreDTO} from "../models/GenreDTO";
+
+export default {
+    title: "Components/GenreSelect",
+    component: GenreSelect,
+} as ComponentMeta<typeof GenreSelect>;
+
+const Template: ComponentStory<typeof GenreSelect> = args => {
+    const [selectedGenres, setSelectedGenres] = useState<GenreDTO[]>([]);
+
+    const genres: GenreDTO[] = [
+        { name: "Crime" },
+        { name: "Documentary" },
+        { name: "Horror" },
+        { name: "Comedy" },
+    ];
+
+    return (
+        <>
+            <pre>{selectedGenres.map(v => v.name).join()}</pre>
+            <GenreSelect
+                {...args}
+                genres={genres}
+                onSelect={setSelectedGenres}
```

Comment:
It's recommended to use a separate handler to pass it to props so we can manage and maintain it:
`onSelect={handleGenreSelect}` with `const handleGenresSelect = (newGenres) => setSelectedSenres(newGenres)` 
something like this

Reply:
Done